doc/barman.1.d/50-show-backup.md

```diff
@@ -36,4 +47,30 @@ Backup 20150828T130001:
     Retention Policy     : not enforced
     Previous Backup      : 20150821T130001
     Next Backup          : - (this is the latest base backup)
+    Root Backup          : 20240814T015504
+    Parent Backup        : 20240814T016504
+    Backup chain size    : 3
+    Children Backup(s)   : 20240814T018515
 ```
+
+> **NOTE:**
+> Depending on the version of your Postgres Server or the type
+> of the backup, the output may vary. For example, "incremental" backups
+> were introduced in Postgres 17, so incremental backups specific fields
+> like "WAL summarizer", "Root Backup", "Parent Backup", "Backup chain size",
+> "Children Backup(s)" will not be present in the output of the previous
+> Postgres versions.
+>
+> Below you can find a list of type-specific fields:
+>
+> `Resource savings`: available for "rsync" and "incremental" backups
+> `Root Backup`, `Parent Backup`, `Backup chain size`: available for 
+> "incremental" backups only
+> `Children Backup(s)`: available for "full" and "incremental" backups
+>
+> Please note that if a backup was created with Barman version 3.10 or
```

Comment:
in this paragraph you are sorta repeating what you said in the first one @andremagui

Reply:
Yeah, but here we need to document the fact about the PostgreSQL version and the Barman version on the output.

Any suggestions on how to organize so we do not repeat documentation here?
@barthisrael @gcalacoci @bleidens @gustabowill

Reply:
is this understandable?
> **NOTE:**
> With the introduction of "incremental" backups in PostgreSQL 17 and in
> Barman 3.11, depending on the version of your Postgres Server, the 
> Barman version and the type of the backup, the output may vary.
>
> Below you can find a list of type-specific fields:
>
> `Resource savings`: available for "rsync" and "incremental" backups
> `Root Backup`, `Parent Backup`, `Backup chain size`: available for 
> "incremental" backups only
> `Children Backup(s)`: available for "full" and "incremental" backups
>
> Please note that if a backup was created with Barman version 3.10 or
> earlier, and you use the `show-backup` command with Barman version 3.11
> or later, the output may not include certain fields. This is also true
> when using the command on a backup created from a Postgres server 
> with version <= 16. Specifically, fields such as `WAL summarizer` and 
> those related to "incremental" backups may be missing from the output
> because they were introduced in version 3.11.